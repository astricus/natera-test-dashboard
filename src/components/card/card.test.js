import React from 'react';
import { shallow } from 'enzyme';
import Card from './card.component';

test('should render Card component', () => {
  expect(shallow(<Card isPost={true}>Post Card</Card>)).toMatchSnapshot();
});
