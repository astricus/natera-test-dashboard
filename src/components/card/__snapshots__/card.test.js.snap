// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Card component 1`] = `
<div
  className="card card--post"
>
  Post Card
</div>
`;
